sdk/cognitivelanguage/cognitivelanguage-conversations/package.json

```
@@ -0,0 +1,94 @@
+{
+  "name": "azure-ai-language-conversations",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for ConversationAnalysisClient.",
+  "version": "1.1.0b2",
```

The version format we use in JS is slightly different from that used in Python:
```suggestion
  "version": "1.0.0-beta.1",
```